rotkehlchen/chain/manager.py

```diff
@@ -226,6 +227,14 @@ def __init__(
                         method=self._initialize_compound,
                         premium=premium,
                     )
+                elif given_module == 'uniswap':
+                    self.eth_modules['uniswap'] = 'loading'
+                    greenlet_manager.spawn_and_track(
```

Comment:
There is no reason to spawn uniswap module asynchronously. We do that in compound only because it has an async call (to get the comptroller address) at its `__init__`. So you can just initialize it normally, like Aave is.